docs/glossary.md

```diff
@@ -132,7 +132,7 @@ OSR (offscreen rendering) can be used for loading heavy page in
 background and then displaying it after (it will be much faster).
 It allows you to render page without showing it on screen.
 
-For more information, read the [Offscreen Rendering][osr] tutorial.
+For more information, read the [Offscreen Rendering] tutorial.
```

Comment:
this should be a link!

Reply:
It is a link. 🙂 At the bottom of the doc there's a reference link for `[offscreen rendering]` (matching is case-insensitive). It was broken before because `[osr]` doesn't exist as a reference link in the doc.

Reply:
Needs to be 
```suggestion
For more information, read the [Offscreen Rendering][] tutorial.
```

then :)

Reply:
It doesn't actually - see [my other comment](https://github.com/electron/electron/pull/36519#discussion_r1037458866) (just dropped it). If you render the preview here on GitHub you can see that it does make it a link as intended.